SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -142,7 +144,7 @@ class UITable extends Component {
         rowData
       );
 
-      return <td key={tdIndex} className={col.props.className}>{cell}</td>;
+      return <td key={col.props.id} className={col.props.className}>{cell}</td>;
```

Comment:
Good point. Yeah, props.id actually needs to be unique for a lot of reasons. I'll put a note in.